examples/cms-builder-io/README.md

```diff
@@ -0,0 +1,99 @@
+# A statically generated blog example using Next.js and Builder.io
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Builder.io](https://builder.io/) as the data source.
+
+## Demo
+
+[https://cms-builder-blog.vercel.app/](https://cms-builder-blog.vercel.app/)
+
+## Deploy your own
```

Comment:
Id move this to the deploy section below, then you can also reomve the note about having the env var setup, as the reader would have gone through the steps and have the env var setup

Reply:
Thanks for the feedback, addressed